packages/dtos/src/job.ts

```diff
@@ -17,8 +17,9 @@ import { Gender, Job, JobStatus, Shooting } from '@modela/database'
 export enum SearchJobStatus {
   'OPEN' = 'OPEN',
   'CLOSE' = 'CLOSE',
+  'CANCELLED' = 'CANCELLED',
+  'REPORTED' = 'REPORTED',
```

Comment:
(Discussion)
This is affected to `/jobs` in swagger also 
(Anyone besides Admin should not query with `REPORTED` or `CANCELLED` ?)

![image](https://user-images.githubusercontent.com/35169079/220868817-68803321-bd4d-489d-a448-aa7cd8dab482.png)

Reply:
hmmm ok, should create separate ENUM for each search type